src/hash/blake/mod.rs

```diff
@@ -0,0 +1,261 @@
+use super::{ElementHasher, HashFn};
+use crate::{
+    ByteReader, ByteWriter, Deserializable, DeserializationError, Digest, Felt, FieldElement,
+    Serializable,
+};
```

Comment:
minor nit: since we import most things from `crate`, we can probably import `ElementHasher` and `HashFn` from there too.

Reply:
Fixed!